src/chainstate/burn/db/sortdb.rs

```diff
@@ -3382,6 +3438,31 @@ impl<'a> SortitionHandleTx<'a> {
             values.push(snapshot.sortition_id.to_hex());
         }
 
+        if let Some(initialize_bonus) = initialize_bonus {
+            // first sortition with a winner, set the initial mining bonus fields
+            keys.push(db_keys::initial_mining_bonus_per_block().into());
+            values.push(initialize_bonus.per_block.to_string());
+
+            let total_reward_remaining = initialize_bonus.total_reward - initialize_bonus.per_block;
```

Comment:
Is there a case where this could be negative?  Should we use a saturating sub to be safe?

Reply:
This couldn't be negative, as `per_block` is set as just `total_reward / INITIAL_MINING_WINDOW`, but I updated to saturating sub just to be on the safe side in ce026c8a1.